experimental/saiswitchextensions.h

```diff
@@ -80,26 +80,11 @@ typedef void (*sai_ha_set_event_notification_fn)(
         _In_ uint32_t count,
         _In_ const sai_ha_set_event_data_t *data);
 
-/**
- * @brief HA scope event type
- */
-typedef enum _sai_ha_scope_event_t
-{
-    /** HA scope state changed */
-    SAI_HA_SCOPE_EVENT_STATE_CHANGED,
-
-    /** Flow reconcile is needed */
-    SAI_HA_SCOPE_EVENT_FLOW_RECONCILE_NEEDED,
-} sai_ha_scope_event_t;
-
 /**
  * @brief Notification data format received from SAI HA scope callback
  */
 typedef struct _sai_ha_scope_event_data_t
 {
-    /** Event type */
-    sai_ha_scope_event_t event_type;
```

Comment:
such changes will not be backward compatible

Reply:
if we put this event data at the end, will this be backward compatible?

Reply:
no xD chanigng struct size will not be backward compatible at all, but if you need to add something, add it at the end